SR2019/2018-10-10-place-confirmation.md

```diff
@@ -0,0 +1,11 @@
+---
+to: Student Robotics 2019
+subject: Confirming your place at Student Robotics 2019
+note: This email will be sent multiple times, to those in the list not in the "Contacted" segment
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2019! If you've applied for a second team, we sadly can't confirm you will receive a second kit until our deadline on the 26th.
+
+The specific details for the London kickstart will be sent over soon to those who have requested the London venue. The London venue is small, and so we're only able to handle a small number of teams. Please only go to this venue if you've been confirmed. The Southampton venue (University of Southampton) will be available to all other teams.
+
```

Comment:
We probably want to say something about teams that can't attend, saying we'll arrange shipping to them after kickstart and mention that they will need to pay for shipping

Reply:
What was that about no cost of entry? Some teams will be unable to attend kickstart through no fault of their own, for example our teams in Germany. This is effectively penalising them and possibly is a barrier to entry.

Reply:
This is a good point. We have always aimed to eliminate implicit cost of entry via things like travel, but this is explicit cost of entry which is basically saying "if you can't make it to kickstart you must pay". We should look into the possibility of funding shipping, as this issue is reasonably likely to discourage teams distant from kickstart location from taking part.

Reply:
Believe that we pay for shipping to teams who can't make kickstart, and teams only pay return postage if they drop out early or they want to hold on to the kits? Would have to check what we've done in the past but shipping costs won't be that high.

Reply:
in the last competition I helped run, shipping was paid for by teams. I'm totally up for us paying for them this year though

Reply:
I suggest we open an issue about this and discuss this properly in a meeting, rather than in the comments of a somewhat-unrelated email PR